tsconfig.json

```diff
@@ -20,22 +20,21 @@
     ],
     "paths": {
       "@/*": ["./*"],
```

Comment:
We should remove this rule it's too wide and causes issue with absolute paths in jest. If a specific path is needed then add it. Otherwise fallback onto the regular relative paths '../../../mything'.

Reply:
Raised: https://nhsd-jira.digital.nhs.uk/browse/CCM-5712